pkg/cloudprovider/aws/utils/retryer.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/client"
+	"github.com/aws/aws-sdk-go/aws/request"
+)
+
+// Retryer implements the aws request.Retryer interface
+// and adds support for retrying ec2 InvalidInstanceID.NotFound
+// which can occur when instances have recently been created
+// and are not yet describe-able due to eventual consistency
+type Retryer struct {
+	baseRetryer request.Retryer
```

Comment:
just inline, imo

type Retryer struct {
 request.Retryer
}

Reply:
ah that is cleaner, I always for get you can ref the embedded type